.gitignore

```diff
@@ -2,7 +2,7 @@ node_modules
 dist
 dist-*
 .memsourcerc.yaml
-docs/.vuepress/public/assets/json/packages.json
+/docs/docs/.vuepress/public/assets/json/packages.json
```

Comment:
`/docs/docs` ? Any reason why having a nested `docs` folder in a folder already named `docs` ?

Reply:
`/docs` has been added as a new unique workspace.
`/docs/docs` is where all `*.md` files are stored (it could be also renamed as `src` but not mandatory).